src/main/java/com/conveyal/r5/transit/TransportNetwork.java

```diff
@@ -345,37 +333,29 @@ public TransportNetwork applyScenario (Scenario scenario) {
 
     /**
      * We want to apply Scenarios to TransportNetworks, yielding a new TransportNetwork without disrupting the original
-     * one. The approach is to make a copy of the TransportNetwork, then apply all the Modifications in the Scenario
-     * one by one to that same copy. Two very different modification strategies are used for the TransitLayer and the
-     * StreetLayer.
-     * The TransitLayer has a hierarchy of collections, from patterns to trips to stoptimes. We can
+     * one. The approach is to make a copy of the TransportNetwork, then apply all the Modifications in the Scenario one
+     * by one to that same copy. Two very different modification strategies are used for the TransitLayer and the
+     * StreetLayer. The TransitLayer has a hierarchy of collections, from patterns to trips to stoptimes. We can
      * selectively copy-on-modify these collections without much impact on performance as long as they don't become too
      * large. This is somewhat inefficient but easy to reason about, considering we allow both additions and deletions.
-     * We don't use clone() here with the expectation that it will be more clear and maintainable to show exactly
-     * how each field is being copied.
-     * On the other hand, the StreetLayer contains a few very large lists which would be wasteful to copy.
-     * It is duplicated in such a way that it wraps the original lists, allowing them to be non-destructively extended.
-     * There will be some performance hit from wrapping these lists, but it's probably completely negligible.
-     * @return a semi-shallow copy of this TransportNetwork.
+     * We don't use clone() here with the expectation that it will be more clear and maintainable to show exactly how
+     * each field is being copied. On the other hand, the StreetLayer contains a few very large lists which would be
+     * wasteful to copy. It is duplicated in such a way that it wraps the original lists, allowing them to be
+     * non-destructively extended. There will be some performance hit from wrapping these lists, but it's probably
+     * negligible.
+     *
+     * @return a copy of this TransportNetwork that is partly shallow and partly deep.
      */
     public TransportNetwork scenarioCopy(Scenario scenario) {
+        // Maybe we should be using clone() here but TransportNetwork has very few fields and most are overwritten.
         TransportNetwork copy = new TransportNetwork();
-        copy.networkId = scenario.id;
```

Comment:
I don't think we'll ever have two network objects with the same ID. This change does not mean we're no longer setting it, the field is just called scenarioId now (here and in the street and transit layers) because it's not always a transit network ID. It's usually a scenario ID, and arguably when a scenario is not applied, the network ID is the ID of the baseline empty scenario.